web/packages/teleterm/src/ui/DocumentAccessRequests/NewRequest/NewRequest.tsx

```diff
@@ -17,86 +17,35 @@
  */
 
 import styled from 'styled-components';
+import { Alert, Box, Link } from 'design';
 
-import { Alert, Box, Flex, Link, Indicator } from 'design';
-import { space, SpaceProps, width } from 'design/system';
-
-import {
-  ShowResources,
-  Cluster,
-} from 'gen-proto-ts/teleport/lib/teleterm/v1/cluster_pb';
-
-import { SearchPagination, SearchPanel } from 'shared/components/Search';
+import { ShowResources } from 'gen-proto-ts/teleport/lib/teleterm/v1/cluster_pb';
 import {
   ResourceList,
   ResourceMap,
 } from 'shared/components/AccessRequests/NewRequest';
 
-import {
-  PendingAccessRequest,
-  extractResourceRequestProperties,
-} from 'teleterm/ui/services/workspacesService/accessRequestsService';
+import { PendingAccessRequest } from 'teleterm/ui/services/workspacesService/accessRequestsService';
 
 import { useWorkspaceContext } from 'teleterm/ui/Documents';
 import { useAppContext } from 'teleterm/ui/appContextProvider';
 
-import useNewRequest, { ResourceKind } from './useNewRequest';
-
-const agentOptions: ResourceOption[] = [
-  { value: 'role', label: 'Roles' },
-  {
-    value: 'node',
-    label: 'Servers',
-  },
-  {
-    value: 'app',
-    label: 'Apps',
-  },
-  {
-    value: 'db',
-    label: 'Databases',
-  },
-  {
-    value: 'kube_cluster',
-    label: 'Kubes',
-  },
-];
-
 export function NewRequest() {
```

Comment:
Let's slap a JSDoc here that this is for requesting roles only, available through the additional actions -> new role request.